src/KubernetesClient/Informers/ResourceStreamType.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace k8s.Informers
+{
+    /// <summary>
+    ///     The type of resource observable stream that specifies whether to return current state of resource, observe changes, or both
+    /// </summary>
+    [Flags]
+    public enum ResourceStreamType
+    {
+        /// <summary>
+        ///     A Cold observable that returns current state of resources and then completes
+        /// </summary>
+        List = 1,
+
+        /// <summary>
+        ///     A Hot observable that publishes a list of changes as they happen
+        /// </summary>
+        Watch = 2,
+
+        /// <summary>
+        ///     A Hot observable that Lists current state of resources followed by watch.
+        /// </summary>
+        ListWatch = 3
```

Comment:
Consider `ListWatch = List | Watch` for clarity.

Reply:
I originally had it that way, however, given that this is public API that developers will use commonly I was trying to avoid forcing people to use bitvise combinations on flags. A lot of developers are actually not familiar with Flags and how to properly use them. ListWatch gives a "simple" option for them to use that is intuitive. This is unlike `EventTypeFlags` which is considered "low level" API and most developers are unlikely to interact with it directly, but even if they do I made sure the naming convention is very clear to that they should be apply HasFlag when using it.

Reply:
I simply meant changing the declaration of the enum member to be:
```csharp
public enum ResourceStreamType
{
    ListWatch = List | Watch
}

```